controller/src/main/java/io/pravega/controller/server/eventProcessor/LocalController.java

```diff
@@ -565,6 +565,12 @@ private StreamSegments getStreamSegments(List<SegmentRange> ranges) {
                 .thenApply(status -> ModelHelper.encode(status.getState(), stream + " " + txnId));
     }
 
+    @Override
+    public CompletableFuture<List<UUID>> listCompletedTxns(Stream stream) {
+        return controller.listCompletedTxns(stream.getScope(), stream.getStreamName(), requestIdGenerator.nextLong())
+                .thenApply(result -> result);
```

Comment:
Line no 572 might not be required as you're not computing anything on result

Reply:
Make the changes.